### DISCUSSION TITLE
Cutscene Head Tracking RE7

### DISCUSSION THREAD

**Discussion by Nikadis** [praydog/REFramework#1048] [General Questions]
Hi guys,
I would like to ask if there is any option in the mod to enable head tracking in cutscenes. In other words, how to unlock the fixed camera during cutscenes. It brings motion sickness.
Thanks

**Comment by Fredhoob** [comment]
I think all you can do is activate the option 'Decoupled camera pitch' under VR in the menu, it will make sure that the camera never moves on the vertical axis, which helps tremendously with comfort and greatly reduce motion sickness.
